advisories/0010.md

```diff
@@ -0,0 +1,49 @@
+### MCR: runc process.cwd and leaked fds container breakout 
+#### Release Date
+2024-01-31
+#### Overview
+Several container breakouts due to internally leaked fds.
+
+Consult the [upstream advisory][upstream] and [snyk][snyk.io] for details, mitigations, workarounds, and patches.
+#### Affected Products
+Mirantis Container Runtime (MCR) <= 23.0.9
+### Vulnerability Information
+#### CVE Identifier
+[CVE-2024-21626][nvd.nist.gov]
+#### CVSSv3.1
+8.6 (High) CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:C/C:H/I:H/A:H
+#### CWEs
+CWE-403, CWE-668
+#### How to check if your environment is vulnerable
```

Comment:
They could also check using `docker version`. We should tell them how.

Reply:
```
docker version --format '{{range .Server.Components}}{{if eq .Name "runc"}}{{.Version}}{{end}}{{end}}'
```

Reply:
Done